OpenRA.Game/Network/Order.cs

```diff
@@ -239,21 +252,27 @@ public static Order CancelProduction(Actor subject, string item, int count)
 
 		// For scripting special powers
 		public Order()
-			: this(null, null, Target.Invalid, null, false, CPos.Zero, 0) { }
+			: this(null, null, Target.Invalid, null, false, null, CPos.Zero, 0) { }
 
 		public Order(string orderString, Actor subject, bool queued)
-			: this(orderString, subject, Target.Invalid, null, queued, CPos.Zero, 0) { }
+			: this(orderString, subject, Target.Invalid, null, queued, null, CPos.Zero, 0) { }
 
 		public Order(string orderString, Actor subject, Target target, bool queued)
-			: this(orderString, subject, target, null, queued, CPos.Zero, 0) { }
+			: this(orderString, subject, target, null, queued, null, CPos.Zero, 0) { }
+
+		public Order(string orderString, Actor subject, Target target, bool queued, List<Actor> extraActors)
+			: this(orderString, subject, target, null, queued, extraActors, CPos.Zero, 0) { }
```

Comment:
Consider adding a unit test for this ctor to the existing unit tests that check Orders can round-trip.

Reply:
I'm not sure how to do that considering the test would need to construct an actual world and actors to be meaningful.

Reply:
Fair.